common/src/main/java/org/terracotta/angela/common/distribution/Distribution107Controller.java

```diff
@@ -408,7 +418,9 @@ public String pluginJarsRootFolderName(Distribution distribution) {
 
     if (server.getSecurityDir() != null) {
       options.add("-x");
-      options.add(server.getSecurityDir().toString());
+      Path securityRootDirectoryPath = workingDir.toPath().resolve("security-root-directory-" + server.getServerSymbolicName().getSymbolicName());
```

Comment:
just to be sure, are the audit- and security-root-directory- fixed names or could they change through the config?

Reply:
It is configurable